src/patient-registration/patient-registration.resource.tsx

```diff
@@ -132,3 +132,51 @@ export function saveRelationship(abortController: AbortController, relationship:
     signal: abortController.signal,
   });
 }
+
+export async function savePatientPhoto(
+  patientUuid: string,
+  file: File,
+  fileCaption: string,
+  abortController: AbortController,
+  base64Content: string,
+  url: string,
+  obsDatetime: string,
+  concept: string,
+) {
+  const formData = new FormData();
+  const json = {};
+  json['person'] = patientUuid;
```

Comment:
These can be initialized directly, i.e.,

```js
const json = {
  person: patientUuid,
  ...
};
```

if you don't want that type then declare it a `Record<string, any>`, such as `const json: Record<string, any> = ...`.